garden_ai/mlmodel.py

```diff
@@ -252,16 +254,32 @@ def stage_model_from_memory(
     if extra_paths is None:
         extra_paths = []
 
+    def validate_extra_path(path: pathlib.Path):
```

Comment:
nope, left it there just because it's such a bespoke helper function it didn't really feel like it warranted a move to `utils`; if it smells funny here though I don't mind pulling it out

Reply:
I don't think your only options are defining it in this context or sending it to utils. It could also be a top level function in this mlmodel module.